helper-cli/src/main/kotlin/commands/MapCopyrightsCommand.kt

```diff
@@ -85,7 +85,7 @@ private fun OrtResult.getUnprocessedCopyrightStatements(processedStatements: Col
         }
     }
 
-    return processedStatements.flatMapTo(mutableSetOf<String>()) {
+    return processedStatements.flatMapTo(mutableSetOf()) {
         if (processedToUnprocessed.containsKey(it)) processedToUnprocessed[it]!! else listOf(it)
```

Comment:
Unrelated, but maybe you feel like changing this line to `processedToUnprocessed.getOrDefault(it, listOf(it))` in a separate commit.

Reply:
Done.